Lib/test/clinic.test.c

```diff
@@ -5464,3 +5396,336 @@ docstr_fallback_to_converter_default(PyObject *module, PyObject *const *args, Py
 static PyObject *
 docstr_fallback_to_converter_default_impl(PyObject *module, str a)
 /*[clinic end generated code: output=ae24a9c6f60ee8a6 input=0cbe6a4d24bc2274]*/
+
+
+/*[clinic input]
+test_deprecate_positional_use_1
+    pos: object
+    * [from 3.14]
+    optarg: int = 5
```

Comment:
For simplicity you can use the `object` converter here.

Reply:
Yes, that's true. I'll update it.